metrics/metric.py

```
@@ -0,0 +1,887 @@
+
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+from retry import retry
+
+
+def fit_topic_pdf(topic, topk=2000, show_img=False, save_img_pth=None):
+    import numpy as np
+    import matplotlib.pyplot as plt
+
+    import numpy as np
```

Ruff (0.8.2)

Redefinition of unused `np` from line 19

Remove definition: `np`

(F811)